packages/video_player_videohole/tizen/src/dart_api_dl.c

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+ * for details. All rights reserved. Use of this source code is governed by a
+ * BSD-style license that can be found in the LICENSE file.
+ */
+
+// This is a hack.
+// This file was copied from flutter-tizen/flutter/bin/cache/dart-sdk/include/dart_api_dl.c.
+// We need to link to this file from tizen_native_plugin.cc.
+// This file caused a build error when flutter-tizen build was used.
+// The workaround was to copy this file here and make changes
+// to make building possible.
+// See FindFunctionPointer() function for details.
+
+// To make this file work we had to change #includes,
+// the original ones are below.
+// #include "dart_api_dl.h"               /* NOLINT */
+// #include "dart_version.h"              /* NOLINT */
+// #include "internal/dart_api_dl_impl.h" /* NOLINT */
+// Here are the modified ones:
+#include <../../../../../dart-sdk/include/dart_api_dl.h>  /* NOLINT */
+#include <../../../../../dart-sdk/include/dart_version.h> /* NOLINT */
+#include <../../../../../dart-sdk/include/internal/dart_api_dl_impl.h> /* NOLINT */
```

Comment:
don't use absolute path, it may cause build break

Reply:
Let me check if we can automatically inject the Dart SDK include path from the flutter-tizen tool during plugin build.

Otherwise you can possibly use the below script to normalize the paths to the headers and add them to `USER_INC_DIRS`.

https://github.com/flutter-tizen/tizen_interop/pull/19#discussion_r1101130957

Reply:
I prepared a patch here: https://github.com/flutter-tizen/flutter-tizen/pull/488

Please take a look.

Reply:
ok, i will change the path